identifiers/country-my/house_of_representatives.csv

```diff
@@ -0,0 +1,232 @@
+ocd-division/country:my/state:10/ed:tanjong_karang,Tanjong Karang,,
+ocd-division/country:my/state:11/ed:besut,Besut,,
+ocd-division/country:my/state:11/ed:dungun,Dungun,,
+ocd-division/country:my/state:11/ed:hulu_terengganu,Hulu Terengganu,,
+ocd-division/country:my/state:11/ed:kemaman,Kemaman,,
+ocd-division/country:my/state:11/ed:kuala_nerus,Kuala Nerus,,
+ocd-division/country:my/state:11/ed:kuala_terengganu,Kuala Terengganu,,
+ocd-division/country:my/state:11/ed:marang,Marang,,
+ocd-division/country:my/state:11/ed:setiu,Setiu,,
+ocd-division/country:my/state:12/ed:batu_sapi,Batu Sapi,,
+ocd-division/country:my/state:12/ed:beaufort,Beaufort,,
+ocd-division/country:my/state:12/ed:beluran,Beluran,,
+ocd-division/country:my/state:12/ed:kalabakan,Kalabakan,,
+ocd-division/country:my/state:12/ed:keningau,Keningau,,
+ocd-division/country:my/state:12/ed:kimanis,Kimanis,,
+ocd-division/country:my/state:12/ed:kinabatangan,Kinabatangan,,
+ocd-division/country:my/state:12/ed:kota_belud,Kota Belud,,
+ocd-division/country:my/state:12/ed:kota_kinabalu,Kota Kinabalu,,
+ocd-division/country:my/state:12/ed:kota_marudu,Kota Marudu,,
+ocd-division/country:my/state:12/ed:kudat,Kudat,,
+ocd-division/country:my/state:12/ed:lahad_datu,Lahad Datu,,
+ocd-division/country:my/state:12/ed:libaran,Libaran,,
+ocd-division/country:my/state:12/ed:papar,Papar,,
+ocd-division/country:my/state:12/ed:penampang,Penampang,,
+ocd-division/country:my/state:12/ed:pensiangan,Pensiangan,,
+ocd-division/country:my/state:12/ed:putatan,Putatan,,
+ocd-division/country:my/state:12/ed:ranau,Ranau,,
+ocd-division/country:my/state:12/ed:sandakan,Sandakan,,
+ocd-division/country:my/state:12/ed:semporna,Semporna,,
+ocd-division/country:my/state:12/ed:sepanggar,Sepanggar,,
+ocd-division/country:my/state:12/ed:silam,Silam,,
+ocd-division/country:my/state:12/ed:sipitang,Sipitang,,
+ocd-division/country:my/state:12/ed:tawau,Tawau,,
+ocd-division/country:my/state:12/ed:tenom,Tenom,,
+ocd-division/country:my/state:12/ed:tuaran,Tuaran,,
+ocd-division/country:my/state:13/ed:bandar_kuching,Bandar Kuching,,
+ocd-division/country:my/state:13/ed:baram,Baram,,
+ocd-division/country:my/state:13/ed:batang_lupar,Batang Lupar,,
+ocd-division/country:my/state:13/ed:batang_sadong,Batang Sadong,,
+ocd-division/country:my/state:13/ed:betong,Betong,,
+ocd-division/country:my/state:13/ed:bintulu,Bintulu,,
+ocd-division/country:my/state:13/ed:hulu_rajang,Hulu Rajang,,
+ocd-division/country:my/state:13/ed:igan,Igan,,
+ocd-division/country:my/state:13/ed:julau,Julau,,
+ocd-division/country:my/state:13/ed:kanowit,Kanowit,,
+ocd-division/country:my/state:13/ed:kapit,Kapit,,
+ocd-division/country:my/state:13/ed:kota_samarahan,Kota Samarahan,,
+ocd-division/country:my/state:13/ed:lanang,Lanang,,
+ocd-division/country:my/state:13/ed:lawas,Lawas,,
+ocd-division/country:my/state:13/ed:limbang,Limbang,,
+ocd-division/country:my/state:13/ed:lubok_antu,Lubok Antu,,
+ocd-division/country:my/state:13/ed:mambong,Mambong,,

```

Comment:
It looks like Mambong was removd in 2016: https://en.wikipedia.org/wiki/Mambong_(federal_constituency)

Reply:
done